server/routes/appointmentRouter.ts

```diff
@@ -91,12 +91,16 @@ export default function appointmentRouter(services: Services): Router {
   get(
     '/api/get-offender-events',
     auditPageAccessAttempt({ services, page: ApiAction.OffenderEvents }),
+    isCreateIndividualAppointmentRolledOut,
+    checkPrisonerInCaseload({ activeCaseloadOnly: true }),
```

Comment:
These API routes should have the same protection as the page that's calling them